lib/generators/doorkeeper/openid_connect/templates/initializer.rb

```diff
@@ -28,6 +28,18 @@
     # redirect_to new_user_session_url
   end
 
+  # Depend on the configuration, the DoubleRenderError could be occurred
+  #  because of render/redirect is called at some configuration before this configure is called.
+  # To avoid DoubleRenderError, you could adding these two lines at the beginning
+  #  of this configuration: (Reference: https://github.com/rails/rails/issues/25106)
```

Comment:
```suggestion
  # Depending on your configuration, a DoubleRenderError could be raised
  # if render/redirect_to is called at some point before this callback is executed.
  # To avoid the DoubleRenderError, you could add these two lines at the beginning
  #  of this callback: (Reference: https://github.com/rails/rails/issues/25106)
```